Python.md

```diff
@@ -40,7 +85,37 @@ result = connection.execute(sqlalchemy.text("""
            show_gravatar
       FROM "user"
      WHERE musicbrainz_id IN :musicbrainz_usernames
-"""), {
-    'musicbrainz_usernames': tuple(usernames),
-})
+""")
+result = connection.execute(query, {'musicbrainz_usernames': tuple(usernames)})
 ```
+
+It is preferred to assign the query to a separate variable and not include it inside the
+`execute` call.
+
+TODO: Projects which don't use sqlalchemy
```

Comment:
out of interest - do we have any python database projects which **don't** use sqlalchemy? Perhaps this todo doesn't exist

- [ ] Metabrain.org
- [ ] CB
- [ ] LB, MeB
- [ ] AB